internal/controllers/reconciliation/controller.go

```diff
@@ -195,23 +160,6 @@ func (c *Controller) Reconcile(ctx context.Context, req reconstitution.Request)
 		ready = status.Ready
 	}
 
-	// Evaluate the readiness of resources in the previous readiness group
```

Comment:
this is all enclosed in op now?

Reply:
It's all handled by the synthesis cache and resource tree concepts so the reconciliation controller doesn't need to worry about the dependency graph.